back/internal/postsvc/internal/postdb/qrypost.go

```diff
@@ -37,14 +38,19 @@ func (s *PostDB) upsertType(ctx cx, sid string, x *pb.AddTypeReq, y *pb.TypeResp
 		return err
 	}
 
-	_, err = stmt.Exec(id, x.name)
+	_, err = stmt.Exec(id, x.Name)
 
 	if err != nil {
 		return err
 	}
 
-	y.id = id
-	y.name = x.name
+	intID, err := strconv.Atoi(id.String())
+	if err != nil {
+		return err
+	}
+
+	y.Id = uint32(intID)
```

Comment:
Is the type juggling to/from uint32/string necessary?

Reply:
I might be wrong, but it seems uidgen library returns string IDs, while `post.proto` requires the IDs be uint32. 

Being that I would prefer not to change the uidgen library, should we update post.proto to require string types for IDs?

Reply:
If everything else is working for now, please leave these alone and file a new issue to swap the ids in the protos to strings (one issue per proto file, please).

Reply:
I've created Issue #58 and Issue #59.